tests/system/model_monitoring/assets/application.py

```diff
@@ -71,5 +73,38 @@ def do_tracking(
         ]
 
 
+class DemoMonitoringAppV2(ModelMonitoringApplicationBaseV2):
```

Comment:
Why not replace the "V1"?

Reply:
We are still need to support V1 until 1.9.0